main.py

```diff
@@ -1,10 +1,60 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def open_player_file() -> dict:
+    with open("players.json") as players:
+        return json.load(players)
+
+
+def create_race(data: dict) -> Race:
+    race, _ = Race.objects.get_or_create(
+        name=data["race"]["name"],
+        description=data["race"]["description"]
+    )
+    return race
+
+
+def create_skill(data: dict) -> None:
+    for skill in data["race"]["skills"]:
+        skill, _ = Skill.objects.get_or_create(
+            name=skill["name"],
+            bonus=skill["bonus"]
+        )
+        skill.race = create_race(data)
```

Comment:
Create race once at the begginning of the function
And then just assign the value in a loop

This way you'll reduce some queries to the database